elide-standalone/src/test/resources/configs/models/tables/SalesView.hjson

```diff
@@ -0,0 +1,75 @@
+{
+  tables:
```

Comment:
Let's move all of this test infrastructure into AggregationStore instead of duplicating it in Spring and Standalone.

Reply:
Removed the tests from Spring and Standalone.